rollup.config.js

```diff
@@ -57,42 +67,24 @@ function createCommonJSConfig(input, output) {
     external,
     plugins: [
       resolve({ extensions }),
-      typescript(),
```

Comment:
Is this because babel can handle (strip) types?
I assume you tried and check the result doesn't change at all.
(Also, please apply it to jotai and valtio, too.)

Reply:
I did yes, it's a bit hard to track exactly due to the naming changes, and the addition of some extra outputs, but here is the diff: https://github.com/aulneau/zustand/commit/0322e8847be71b10c7a0a1c5657e3a53d0bd4758

Reply:
Oh, I see > naming changes.
Thanks for confirming.